deploy/kubeturbo/templates/deployment.yaml

```diff
@@ -24,6 +24,10 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       serviceAccountName: turbo-user
+{{- if .Values.image.imagePullSecret }}
+      imagePullSecrets:
+        - name: {{ .Values.image.imagePullSecret }}
+{{- end }}
       containers:
         - name: {{ .Chart.Name }}
```

Comment:
Probably add a section with env variable for passing the kubeturbo namespace via the downward api also:
(Listed is section 4 and 5 [here](https://github.com/turbonomic/kubeturbo/wiki/Yaml-Deployment-Details)
```
          env:
            - name: KUBETURBO_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
```
This will ensure that the jobs are created in the kubeturbo's namespace, in place of namespace `default` when this is missing.

Reply:
thank you - adding this to my commit